src/controllers/HTMLController.ts

```diff
@@ -1,14 +1,23 @@
+import express from 'express'
 import { Readable } from 'node:stream'
-import { Controller } from 'tsoa'
+import { Controller, Middlewares } from 'tsoa'
 
 export type HTML = Readable
 
 export abstract class HTMLController extends Controller {
+  @Middlewares(async (req: express.Request, _res: express.Response, next: express.NextFunction) => {
+    req.log.trace('returning text/html content %o', { req })
+    next()
+  })
```

Comment:
this looks ... odd. What is this doing? I'm especially confused in that this is an abstract class that doesn't interact with tsoa